### PR TITLE
fix(VCalendar): proper week height in month view on Firefox

### LINKED ISSUE #8319: [Bug Report][V-Calendar] [2.0.4]  "event-more" = true ignored in Firefox
### Environment
**Vuetify Version:** 2.0.4
**Vue Version:** 2.6.10
**Browsers:** Firefox 68.0
**OS:** Ubuntu undefined

### Steps to reproduce
Add more events than can fit to day in monthly calendar view.

### Expected Behavior
Events that fit to calendar view are rendered, rest are hidden and "More events" label/button is rendered (Open the example in Chrome)

### Actual Behavior
All events are rendered, "event-more=true" prop is ignored.

### Reproduction Link
<a href="https://codepen.io/anon/pen/pMpKWM" target="_blank">https://codepen.io/anon/pen/pMpKWM</a>

### PR DESCRIPTION
**Pull request by igadmile**
## Description
As per specification, flex child elements should have `min-height:0` (min-width for rows) to
prevent element overflow. See https://bugzilla.mozilla.org/show_bug.cgi?id=1114904

Without this properties, the `v-calendar-weekly__week` expands to include all event elements and calendar overflows parent element. In cases where there are a lot of events, and `event-more="true"` is set, `updateEventVisibility` method in the `calendar-with-events` mixin can't properly hide events.

## Motivation and Context

Fixes #8319

## How Has This Been Tested?

visually

## Markup:

<details>

```vue
<template>
  <v-sheet height="600">
    <v-calendar
      v-model="focus"
      color="primary"
      :events="events"
      :event-margin-bottom="3"
      :now="today"
      type="month"
    ></v-calendar>
  </v-sheet>
</template>

<script>
export default {
  data: () => ({
    today: "2019-01-08",
    focus: "2019-01-08",
    events: [
      {
        name: "event 1",
        start: "2019-01-14 18:00",
        end: "2019-01-14 19:00",
        color: "#4285F4"
      },
      {
        name: "event 2",
        start: "2019-01-14 18:00",
        end: "2019-01-14 19:00",
        color: "#4285F4"
      },
      {
        name: "event 5",
        start: "2019-01-14 18:00",
        end: "2019-01-14 19:00",
        color: "#4285F4"
      },
      {
        name: "event 3",
        start: "2019-01-14 18:30",
        end: "2019-01-14 20:30",
        color: "#4285F4"
      },
      {
        name: "event 4",
        start: "2019-01-14 19:00",
        end: "2019-01-14 20:00",
        color: "#4285F4"
      },
      {
        name: "event 6",
        start: "2019-01-14 21:00",
        end: "2019-01-14 23:00",
        color: "#4285F4"
      },
      {
        name: "event 7",
        start: "2019-01-14 22:00",
        end: "2019-01-14 23:00",
        color: "#4285F4"
      },
      {
        name: "event 8",
        start: "2019-01-14 22:00",
        end: "2019-01-14 23:00",
        color: "#4285F4"
      },
      {
        name: "event 9",
        start: "2019-01-14 22:00",
        end: "2019-01-15 23:00",
        color: "#4285F4"
      },
      {
        name: "event 10",
        start: "2019-01-14 22:00",
        end: "2019-01-15 23:00",
        color: "#4285F4"
      }
    ]
  })
};
</script>
```
</details>

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:

- [X] The PR title is no longer than 64 characters.
- [X] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [X] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)
- [ ] I've added new examples to the kitchen (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/src/components/VCalendar/VCalendarWeekly.sass [hunk 1]
```diff
@@ -28,6 +28,7 @@
   height: 100%
   display: flex
   flex-direction: column
+  min-height: 0
```
Comment (sh7dm) [inline packages/vuetify/src/components/VCalendar/VCalendarWeekly.sass]: please add bug & issue references here

File: packages/vuetify/src/components/VCalendar/VCalendarWeekly.sass [hunk 2]
```diff
@@ -48,6 +49,7 @@
   display: flex
   flex: 1
   height: 0
+  min-height: 0
```
Comment (sh7dm) [inline packages/vuetify/src/components/VCalendar/VCalendarWeekly.sass]: please add bug & issue references here

File: packages/vuetify/src/components/VCalendar/VCalendarWeekly.sass (hunk 1 above)
Comment (igadmile) [reply packages/vuetify/src/components/VCalendar/VCalendarWeekly.sass]: Added now

File: packages/vuetify/src/components/VCalendar/VCalendarWeekly.sass (hunk 2 above)
Comment (igadmile) [reply packages/vuetify/src/components/VCalendar/VCalendarWeekly.sass]: Added now

**Pull request merged**